Sources/App/Controllers/API/API+PackageCollectionController.swift

```diff
@@ -20,10 +20,10 @@ extension API {
 
     struct PackageCollectionController {
 
-        func generate(req: Request) throws -> EventLoopFuture<PackageCollection> {
+        func generate(req: Request) throws -> EventLoopFuture<SignedCollection> {
```

Comment:
I'm just changing collections over to signed. I was considering making this controllable via an env variable so we could switch signing on and off easily. However, the return type needs to be one or the other.

What we can do is make it `Data` or `String`, i.e. pre-render inside the method. The result for the consumer would be exactly the same. I just didn't want to have to deal with the content-type headers etc so I've stuck with the simple solution which we know will work.